.circleci/config.yml

```diff
@@ -4,14 +4,161 @@ orbs:
   aws-ecr: circleci/aws-ecr@8.1.3
   aws-ecs: circleci/aws-ecs@3.2.0
 
+executors:
+  custom:
+    machine:
+      image: ubuntu-2404:2024.05.1
+    resource_class: xlarge
+
+jobs:
+  build:
+    docker:
+      - image: cimg/node:20.9.0
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - restore_cache:
+          name: "Restore Package Cache"
+          keys:
+            - pnpm-packages-{{ checksum "pnpm-lock.yaml" }}
+      - run:
+          name: "Run build"
+          command: |
+            npm run build
+  code-checks:
```

Comment:
would rather not do this as it is not actionable.  it's not our code.  if it doesn't build, it'll get caught by the build.  If tests fail, they're not our problem.  we rely on the upstream being nice.
this isn't a normal project - it's just a third party image pushing to our docker repo
on top of that, this isn't vital.  only reason i want it to trigger service updates is because the docker images will expire if it doesn't.